retrieval/series_cache.go

```diff
@@ -461,23 +461,9 @@ func (c *seriesCache) lookup(ctx context.Context, ref uint64) (retErr error) {
 			ts.ValueType = meta.ValueType
 		}
 	case textparse.MetricTypeSummary:
-		switch suffix {
-		case metricSuffixSum:
-			ts.Kind = config.CUMULATIVE
-			ts.ValueType = config.DOUBLE
-		case metricSuffixCount:
-			ts.Kind = config.CUMULATIVE
-			ts.ValueType = config.INT64
-		case "": // Actual quantiles.
-			ts.Kind = config.GAUGE
-			ts.ValueType = config.DOUBLE
-		default:
-			// Note: this branch has been seen for a
-			// _bucket suffix, indicating a mix of
-			// histogram and summary conventions.
-			failedReason = "invalid_suffix"
-			return errors.Errorf("unexpected summary suffix %q", suffix)
-		}
+		ts.Name = c.getMetricName(c.metricsPrefix, baseMetricName)
+		ts.Kind = config.CUMULATIVE
+		ts.ValueType = config.DOUBLE
```

Comment:
Summary uses ValueType=DISTRIBUTION, while I suspect we need to use DOUBLE for it. Is this correct?

Reply:
The Kind/ValueType distinction in StackDriver vs. OTLP is very very very confusing. It doesn't really matter in this case, since Summary points have a dedicated OTLP point kind.

Reply:
(there is room for a whole PR to clean this up, since OTLP is different than SD; the ability to select static metadata would be better if it matched OTLP's terms!)